projects/js-packages/publicize-components/src/hooks/use-media-restrictions/index.js

```diff
@@ -28,7 +28,7 @@ const reduceVideoLimits = ( prev, current ) => ( {
 const getVideoLimits = enabledConnections =>
 	enabledConnections
 		.map( connection =>
-			RESTRICTIONS[ connection.service_name ]
+			RESTRICTIONS[ connection.service_name ]?.video
 				? RESTRICTIONS[ connection.service_name ].video
 				: DEFAULT_RESTRICTIONS.video
```

Comment:
Since we have used optional chaining here, we can simplify it as a one-liner
```suggestion
			RESTRICTIONS[ connection.service_name ]?.video || DEFAULT_RESTRICTIONS.video
```

Reply:
Good point!